main.tex

```diff
@@ -39,30 +39,30 @@
+Chaumian CoinJoin~\cite{mizrahi2013blind,maxwell2013coinjoin,zerolink} is a privacy enhancing technique that uses this atomicity property and Chaumian blind signatures~\cite{chaum1983blind} to construct collaborative Bitcoin transactions, also known as CoinJoins. Participants connect to a server, known as the coordinator, and submit their inputs and outputs using different anonymity network identities. That alone provides anonymity but since outputs are unconstrained it's not robust against malicious users who may disrupt the protocol by claiming more than their fair share. To mitigate this the coordinator provides blind signatures representing units of standard denominations in response to submitted inputs. By unblinding and presenting this valid signature back to the coordinator, participants authorize registration of their outputs so the coordinator is unable to link the signed output to a specific input.
 
-The use of standard denominations in the resulting CoinJoin transaction obscures the relationship between individual inputs and outputs, making the origins of each output ambiguous. Unfortunately standard denominations also present limitations, such as requiring change outputs from these CoinJoin transactions, and limiting the usefulness of the privacy enhanced outputs for payments of arbitrary amounts.
+The use of standard denominations in the resulting CoinJoin transaction obscures the relationship between individual inputs and outputs, making the origins of each output ambiguous. Unfortunately standard denominations limits the use of privacy-enhanced outputs for payments of arbitrary amounts and results in change which maintains a link to the non-private input.
 
 \subsection{Limitations of Wasabi}
-In this work, we are aiming to improve on ZeroLink~\cite{zerolink} as implemented by Wasabi, the most popular Chaumian CoinJoin implementation for Bitcoin. We identify several privacy shortcomings and inefficiencies of Wasabi CoinJoins. Some metrics comparing Wasabi, Samourai and other apparent CoinJoin transactions are provided. ``Other'' includes JoinMarket, but also has an inherent false positive error given these transactions are identified heuristically.
+In this work, we aim to improve on ZeroLink~\cite{zerolink} as implemented by Wasabi, the most popular Chaumian CoinJoin implementation for Bitcoin. We identify several privacy shortcomings and inefficiencies of Wasabi CoinJoins. Some metrics comparing Wasabi, Samourai and other apparent CoinJoin transactions are provided. The ``Other'' category includes JoinMarket, but also has an inherent false positive error given these transactions are identified heuristically.
 
-\subsubsection{Round denominations}
+\subsubsection{Fixed denominations}
```

Comment:
FWIW, I think this previously had "fixed" in the title, and I changed it because I felt like that implies it might be fixed between rounds. I don't feel very strongly either way but something to consider

Reply:
Hmmm I read  "round denominations" to mean Round numbers https://en.bitcoin.it/wiki/Privacy#Round_numbers but I see what you mean as fixed between rounds. Maybe uniform denomination? I chose fixed cause that's what was described in the paragraph it titles

Reply:
when in doubt, delete

```suggestion
\subsubsection{Denominations}
```

Reply:
Fixed denomination is incorrect. Not only it is incorrect inter-rounds, but also intra-rounds as the announced denomination almost always differs from the denomination that the final TX will eventually have.  
So fixed denomination is incorrect, and round denomination is misleading. Then let's delete.